lua/muren/ui.lua

```diff
@@ -412,14 +460,13 @@ M.open = function(opts)
   end
   populate_options_buf()
 
-  local gheight, gwidth = get_nvim_ui_size()
-
+  local positions = make_ui_positions()
```

Comment:
(forgot to change this to match the other local var)

I think we could technically also call this only once and pass it around as an argument to the function calls that need it, but I thought there might be some advantage to calling it multiple times that's not obvious, maybe if the values need to be updated if the ui window gets moved during resizing, etc.

Reply:
Yeah I think indeed it's good to compute every time exactly since the window size may change in between :+1: